sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md

```diff
@@ -10,12 +10,17 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
     <dependency>
         <groupId>io.opentelemetry</groupId>
         <artifactId>opentelemetry-sdk</artifactId>
-        <version>0.2.0-SNAPSHOT</version>
+        <version>0.2.0</version>
     </dependency>
     <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-security-keyvault-secrets</artifactId>
-        <version>4.0.0</version>
+        <version>4.1.0-beta.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
```

Comment:
@g2vinay Can you confirm if this dependency needs to be explicitly added for using KV secrets package?

Reply:
Yeah, KV doesn't take compile dependency on Identity.
All the track 2 SDKs, should need to have this explicitly, if they're using Identity on auth.